src/imitation/scripts/config/train_adversarial.py

```diff
@@ -59,30 +70,25 @@ def aliases_default_gen_batch_size(gen_batch_size):
     # the same number is equivalent to not using a replay buffer at all.
     # "Disabling" the replay buffer seems to improve convergence speed, but may
     # come at a cost of stability.
+
     gen_replay_buffer_size = gen_batch_size  # Num generator transitions stored
 
 
 @train_ex.config
-def apply_init_trainer_kwargs_aliases(
+def apply_init_algorithm_kwargs_aliases(
```

Comment:
What's the purpose of this? Do we need two sets of configuration parameters that can be independently varied (they're not really aliases -- I could override `algorithm_kwargs.disc_minibatch_size` to be set to something differently to `disc_minibatch_size`)? As far as I can see `disc_minibatch_size`, `gen_replay_buffer_size` and `disc_batch_size` at the top-level are not used anywhere other than in this config.

I'd suggest either deleting the top-level entries and just keeping `algorithm_kwargs` around, or delete `algorithm_kwargs` and have `train()` pass the relevant top-level entries through to the algorithm directly.

Reply:
I don't remember exactly why I had set it up like this. Your suggestion makes sense, I got rid of the top-level entries and moved them to algorithm_kwargs.